samples/add_portgroup_to_vswitch.py

```diff
@@ -55,17 +56,38 @@ def get_args():
                         action='store',
                         help='Vlan ID')
 
+    parser.add_argument('-c', '--skip_verification',
+                        required=False,
+                        action='store_true',
+                        help='Skip SSL verification')
+
+    parser.add_argument('-r', '--regex_esxi',
+                        required=False,
+                        default=None,
+                        action='store',
+                        help='Regex esxi name')
+
     args = parser.parse_args()
     return args
 
 
-def GetVMHosts(content):
+def GetVMHosts(content, regex_esxi):
```

Comment:
The method parameter should have None as the default value:
def GetVMHosts(content, regex_esxi=None):